src/cs/GameLoop.cs

```diff
@@ -83,6 +83,12 @@ public enum GameState { NOT_STARTED, PLAYING, ENDED };
 		BBs.Add(GetNode<BuildButton>("World/BuildButton2"));
 		BBs.Add(GetNode<BuildButton>("World/BuildButton3"));
 		BBs.Add(GetNode<BuildButton>("World/BuildButton4"));
+		BBs.Add(GetNode<BuildButton>("World/BuildButton5"));
+		BBs.Add(GetNode<BuildButton>("World/BuildButton6"));
+		BBs.Add(GetNode<BuildButton>("World/BuildButton7"));
+		BBs.Add(GetNode<BuildButton>("World/BuildButton8"));
+		BBs.Add(GetNode<BuildButton>("World/BuildButton9"));
+		BBs.Add(GetNode<BuildButton>("World/BuildButton10"));
```

Comment:
I really need to find a better solution for this, it's already getting out of hand ^^"

Reply:
Maybe you could try something like GetNodesInGroup and assign BuildButton to a group.

Reply:
Ok I'll look into it